1-exercises/A-accessing-values/exercise2.js

```diff
@@ -7,7 +7,7 @@
 let capitalCities = {
   UnitedKingdom: "London",
   China: "Beijing",
-  Peru: "Lima"
+  Peru: "Lima",
```

Comment:
While trailing commas don't make a difference to your object definition, they have [their pros and cons](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#:~:text=JavaScript%20has%20allowed%20trailing%20commas,however,%20disallows%20all%20trailing%20commas.). In the end it's often down to the JS style guide your project will use, but it's good to understand the consequences

Reply:
Here (and below) for instance, you probably don't want to add them, because not only they weren't there in the first place and there's probably a reason, but more importantly they introduce changes in your PR that aren't related to what the PR is about. So it makes the "diff" longer by adding unrelated changes.